testsuite/testsuite-microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/SubjectExposingResource.java

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Looks like this resource should be part of the test JAX-RS Application ?

Reply:
I'm not sure I understand, it's used here: https://github.com/thorntail/thorntail/pull/1071/files#diff-528eab1afb2045a3318c303d74d58aabR49

Reply:
Sorry, my fault, I thought for a moment it was part of the fraction source...